pageserver/benches/bench_ingest.rs

```diff
@@ -244,13 +244,7 @@ fn criterion_benchmark(c: &mut Criterion) {
     ];
     let exploded_parameters = {
         let mut out = Vec::new();
-        for io_mode in [
-            IoMode::Buffered,
-            #[cfg(target_os = "linux")]
-            IoMode::Direct,
-            #[cfg(target_os = "linux")]
-            IoMode::DirectRw,
-        ] {
+        for io_mode in [IoMode::Buffered, IoMode::Direct, IoMode::DirectRw] {
```

Comment:
nit: could also just derive `strum_macros::EnumIter` on `IoMode` so it'll always cover all variants.

Reply:
https://github.com/neondatabase/neon/pull/11868/commits/9f1f9912ffe0d6a02cf19196465b9c401bc7420a